README.md

```diff
@@ -101,3 +101,18 @@ We are working on a [markdown version](https://github.com/glpi-project/doc)
  * [TinyMCE v4.7.1](https://github.com/tinymce/tinymce) LGPL2.1
  * [Underscore.js v1.8.3](https://github.com/jashkenas/underscore) MIT
  * [Lodash v unknown](https://github.com/lodash/lodash) MIT
+
+
+## Support
+GLPI is a living software : continuoulsy imporvements are made, new functionnality developped and issues fixed!
```

Comment:
A couple typos exist on this line, and I believe it can be worded better.
Typos were continuoulsy -> continuously, imporvments -> improvements, functionnality -> functionality, and developped -> developed.

Can it be re-worded as follows?
GLPI is a living software. Improvements are continuously made, new functionality is being developed, and issues are being fixed.

Reply:
Thanks !